assignment09/part02/index.php

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<head>
+    <title>Assignment 9, Part 2</title>
+</head>
+<body>
+    
+    <?php
+        //Task 3.1: Change these items to reflect
+        //four of your favorite things
+        $my_favorite_things = array (
+            "1. My first favorite thing is raspberries.",
+            "2. My second favorite thing is film.",
+            "3. My third favorite thing is driving.",
+            "4. My fourth favorite thing is bowling."
```

Comment:
ideally this would just be a list of your favorite thing sentences and we'd let the loop below handle the numbering. That way if the order or length of the array changes - our code still handles it gracefully.